src/System.Net.Sockets/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -13,7 +13,7 @@ namespace System.Net.Sockets.Tests
     public class LoggingTest : RemoteExecutorTestBase
     {
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "NetEventSource is only part of .NET Core")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.Uap, "NetEventSource is only part of .NET Core and requires internal Reflection.")]
```

Comment:
I think the SkipOnTargetFramework attribute should be left alone.  This test will never run on .NET Framework.

However, skipping the test on Uap is temporary and will be solved as part of getting NetEventSource working on Uap. So, for now, you should add an ActiveIssue attribute for that and leave the SkipOnTargetFramework attribute alone.

Comment applies to similar changes below.